alpine-consul/root/usr/bin/consul-join

```diff
@@ -1,16 +1,13 @@
 #!/usr/bin/with-contenv sh
 
-# Get the IP of the current container. We don't want this container to match itself.
-HOST_IP=`container-ip`
+# Get the consul IPs
+JOIN=`container-ip consul | awk '{ printf " -retry-join " $0}'`
```

Comment:
I think `consul-join`:

```
 #!/usr/bin/with-contenv sh
 
-# Get the IP of the current container. We don't want this container to match itself.
-HOST_IP=`container-ip`
+# Get the consul IPs
+JOIN=`container-ip consul | awk '{ printf $0}'`
```

and `run` scripts should have the flag back in-place:

```
s6-setuidgid consul consul agent -config-dir /etc/consul.d -client $BIND -retry-join $JOIN -retry-interval 5s -ui
```